ports/models.py

```diff
@@ -62,7 +62,7 @@ class Builder(models.Model):
 class BuildHistory(models.Model):
     builder_name = models.ForeignKey(Builder, on_delete=models.CASCADE, db_index=True)
     build_id = models.IntegerField()
-    status = models.CharField(max_length=50)
+    status = models.CharField(max_length=50, db_index=True)
     port_name = models.CharField(max_length=50,db_index=True)
     time_start = models.CharField(max_length=50)
```

Comment:
You have two serious database issues here.
* You are ordering builds based on `time_start`, but you lack the index on the field.
* You should store this as a timestamp rather than as a char field.

Reply:
Earlier I was storing the unix time in database and converting it in the views.

Now, changed to storing the normal time.